src/controllers/user.controller.js

```diff
@@ -0,0 +1,42 @@
+import db from '../models/index.js';
+const users = db['users']
+
+export const getAllUsers = async (req, res) => {
+ 
+	try{
+			users.findAndCountAll().then(users => {
+				
+				return res.status(200).json({
+					status:"success",
+					data:users,
+					message:"Retrieved"});
+			})
+		
+	}
+	catch(error) {
+		return res.status(500).json(error.message);
+	}
+	
+};
+
```

Comment:
this is code that should be on the `user.service.js` file. Code that interact with an external service is in the services not controller